runtime/onert/backend/cpu/ops/FullyConnectedLayer.cc

```diff
@@ -177,6 +179,41 @@ void FullyConnectedLayer::fullyConnectedSparseWeight()
     throw std::runtime_error{"FullyConnected: unsupported sparsity"};
 }
 
+void FullyConnectedLayer::fullyConnectedGGMLWeight()
+{
+  if (_bias)
+    throw std::runtime_error{"FullyConnected: GGML weights format does not support bias yet."};
+
+  // convert tensor
+  auto input = getGGMLTensor(_input);
+  auto weights = getGGMLTensor(_weights);
+  auto output = getGGMLTensor(_output);
+  {
+    output.op = GGML_OP_MUL_MAT;
+    output.src[0] = &weights;
+    output.src[1] = &input;
+  }
+  auto *nodes = &output;
+
+  // create graph
+  struct ggml_cgraph graph;
+  {
+    memset(&graph, 0, sizeof(graph));
+    graph.n_nodes = 1;
+    graph.nodes = &nodes;
+  }
+
+  // get cplan
+  auto cplan = ggml_graph_plan(&graph, _external_context->ruy_context()->max_num_threads());
```

Comment:
(Out of curiosity)
Does ggml use ruy thread here?

Reply:
I used this because external context does not support thread num getter yet. I'll update external context class to support thread num getter.